anda/lib/mesa/mesa.spec

```diff
@@ -259,7 +260,7 @@ Requires:       %{name}-filesystem%{?_isa} = %{?epoch:%{epoch}:}%{version}-%{rel
 %if 0%{?with_va}
 Recommends:     %{name}-va-drivers%{?_isa}
 %endif
-Obsoletes:      %{name}-libglapi < 25.0.0~rc2-1
```

Comment:
when in doubt just write `1:25.0.0~rc2-1` directly

Reply:
do this and i approve